### ISSUE TITLE
Add community types/templates to back-end

### ISSUE THREAD

**Issue by tibetsprague** [Hylozoic/hylo-node#512]
Add a new table for community_types. This table should have the fields:
- id
- name
- display_name
- created_at
- updated_at

Add an optional community_type_id to the communities table

**Comment by thomasgwatson** [comment]
Is this ticket relevant anymore? We no longer have 'communities', right?

**Comment by tibetsprague** [comment]
Good point, I did do work for this back in the day, which… could be useful? Templates becomes types? We still probably want to sort out the naming of some of this stuff.

https://github.com/Hylozoic/hylo-node/pull/518

Probably this ticket is not worth keeping, but lemme bring the branch up to date and we can see whats on it.
